### ISSUE TITLE
Implement optional parameters

### ISSUE THREAD

**Issue by SlowMo24** [ypid/pyopening_hours#4]
Is there any plan or even roadmap to implement the optional parameters for the simple API?

**Comment by ypid** [comment]
No. I have currently no need for this so I am not even sure I want to maintain/review PRs. Sorry.
